### PR TITLE
Updated Goodreads

### PR DESCRIPTION
**Pull request by fbernhart**
goodreads

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/goodreads.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...9-.5059 3.2828 0 1.199.141 2.326.425 3.3819.286 1.057.737 1.976 1.368 2.7619.631.78 1.4119 1.397 2.3748 1.833.961.4359 2.119.6609 3.4709.6609 1.248 0 2.3299-.315 3.2619-.946.932-.631 1.6379-1.473 2.1189-2.5249h.061v2.284c0 2.0438-.421 3.6068-1.264 4.7047-.84 1.081-2.2239 1.638-4.1458 1.638-.572 0-1.128-.061-1.669-.181a5.3571 5.3571 0 01-1.4869-.57c-.437-.271-.827-.601-1.143-1.038-.316-.435-.5259-.961-.6319-1.5929h-2.107c.067.887.315 1.654.737 2.3.424.6459.961 1.1719 1.602 1.5928.641.406 1.367.706 2.1719.902.811.194 1.639.3 2.4939.3 1.383 0 2.5409-.195 3.4859-.555.947-.376 1.7139-.902 2.3009-1.608.601-.7079 1.021-1.5488 1.2929-2.5558.27-1.007.42-2.134.42-3.3669zm-7.4837-.557c-.955 0-1.784-.189-2.4789-.571-.697-.3799-1.277-.8819-1.732-1.5029-.467-.621-.7969-1.3319-1.0219-2.1389-.225-.807-.332-1.6329-.332-2.4839 0-.871.105-1.7249.301-2.5629.21-.84.54-1.5869.992-2.2399.451-.652 1.037-1.182 1.7279-1.5839.691-.402 1.533-.605 2.5099-.605.977 0 1.803.209 2.4949.621.676.415 1.247.959 1.683 1.634.436.677.7509 1.4289.9469 2.2548.195.826.285 1.656.285 2.482 0 .8519-.12 1.6779-.345 2.4838-.226.807-.572 1.518-1.038 2.139-.465.621-1.021 1.1229-1.6979 1.5029-.676.382-1.458.571-2.3589.571z"/></svg>
```
Comment (adamrusted) [inline icons/goodreads.svg:1]: Using [this SVG](https://s.gr-assets.com/assets/layout/header/goodreads_logo.svg), it appears their logo may have changed somewhat. Are you able to take another look - and regenerate at 3 decimals please? 🙏🏻

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/goodreads.svg (hunk 1 above)
Comment (fbernhart) [reply icons/goodreads.svg:1]: I don't think the logo has actually changed. Visiting the source https://www.goodreads.com/about/press you can find the PNG of the logo there (unfortunately not as SVG). They use a different "g" for the logo and the full wordmark.

The thicker "g" alone as well can be seen used in the favicon, all of their social media presence and the Google/Apple app stores.

Quickly comparing our current SVG with the PNG at source, there seem to be some small differences. So I'll try to rebuild it from that PNG.

Comment (fbernhart) [reply icons/goodreads.svg]: Done. I've as well changed the color to `#372213`. The "thicker g" in the PNG has different shades of brown, so it's difficult to pick one. But the SVG you've linked about has `#372213` as color. So I would say that it still represents the company pretty good.

goodreads

**Review by adamrusted — APPROVED** [review]
LM! 👍🏻📚

**Pull request merged**